### PR TITLE
on accept input, call terminal `acceptInput`

### LINKED ISSUE #232966: Terminal chat cmd/alt+enter doesn't work on mac
cmd/alt+enter doesn't work in this state on mac:

Image

cmd+enter:

```
2024-11-04 05:11:44.840 [info] [KeybindingService]: / Soft dispatching keyboard event
2024-11-04 05:11:44.841 [info] [KeybindingService]: \ Keyboard event cannot be dispatched
2024-11-04 05:11:44.841 [info] [KeybindingService]: / Received  keydown event - modifiers: [meta], code: MetaLeft, keyCode: 91, key: Meta
2024-11-04 05:11:44.841 [info] [KeybindingService]: | Converted keydown event - modifiers: [meta], code: MetaLeft, keyCode: 57 ('Meta')
2024-11-04 05:11:44.841 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2024-11-04 05:11:44.962 [info] [KeybindingService]: / Soft dispatching keyboard event
2024-11-04 05:11:44.962 [info] [KeybindingService]: | Resolving meta+[Enter]
2024-11-04 05:11:44.963 [info] [KeybindingService]: \ From 23 keybinding entries, no when clauses matched the context.
2024-11-04 05:11:44.977 [info] [KeybindingService]: / Received  keydown event - modifiers: [meta], code: Enter, keyCode: 13, key: Enter
2024-11-04 05:11:44.977 [info] [KeybindingService]: | Converted keydown event - modifiers: [meta], code: Enter, keyCode: 3 ('Enter')
2024-11-04 05:11:44.978 [info] [KeybindingService]: | Resolving meta+[Enter]
2024-11-04 05:11:44.978 [info] [KeybindingService]: \ From 23 keybinding entries, no when clauses matched the context.
2024-11-04 05:11:45.180 [info] [KeybindingService]: + Storing single modifier for possible chord meta.
2024-11-04 05:11:45.482 [info] [KeybindingService]: + Clearing single modifier due to 300ms elapsed.
```

alt+enter:

```
2024-11-04 05:12:01.051 [info] [KeybindingService]: / Soft dispatching keyboard event
2024-11-04 05:12:01.051 [info] [KeybindingService]: \ Keyboard event cannot be dispatched
2024-11-04 05:12:01.052 [info] [KeybindingService]: / Received  keydown event - modifiers: [alt], code: AltLeft, keyCode: 18, key: Alt
2024-11-04 05:12:01.052 [info] [KeybindingService]: | Converted keydown event - modifiers: [alt], code: AltLeft, keyCode: 6 ('Alt')
2024-11-04 05:12:01.052 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2024-11-04 05:12:01.156 [info] [KeybindingService]: / Soft dispatching keyboard event
2024-11-04 05:12:01.170 [info] [KeybindingService]: | Resolving alt+[Enter]
2024-11-04 05:12:01.170 [info] [KeybindingService]: \ From 8 keybinding entries, no when clauses matched the context.
2024-11-04 05:12:01.170 [info] [KeybindingService]: / Received  keydown event - modifiers: [alt], code: Enter, keyCode: 13, key: Enter
2024-11-04 05:12:01.170 [info] [KeybindingService]: | Converted keydown event - modifiers: [alt], code: Enter, keyCode: 3 ('Enter')
2024-11-04 05:12:01.170 [info] [KeybindingService]: | Resolving alt+[Enter]
2024-11-04 05:12:01.170 [info] [KeybindingService]: \ From 8 keybinding entries, no when clauses matched the context.
2024-11-04 05:12:01.358 [info] [KeybindingService]: + Ignoring single modifier alt due to it being pressed together with other keys.
```

### PR DESCRIPTION
**Pull request by meganrogge**
fix #232966

### CODE REVIEW

**Comment by Tyriar** [comment]
Hopefully I resolved the conflict correctly.

File: src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatWidget.ts [hunk 1]
```diff
@@ -139,7 +140,16 @@ export class TerminalChatWidget extends Disposable {
 				}
 			},
 		);
-		this._register(this._inlineChatWidget.chatWidget.onDidChangeViewModel(() => this._saveViewState()));
+		this._register(autorun(r => {
```
Comment (Tyriar) [inline src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatWidget.ts:143]: I think we can delete `acceptInput` now?

Comment (Tyriar) [inline src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatWidget.ts:143]: I'm seeing the old answer come back sometimes now.

Repro:

1. cmd+i
2. list files
3. cmd+enter
4. cmd+i, 🐛 list files Q&A exists

**Review by Tyriar — CHANGES_REQUESTED** [review]

File: src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatWidget.ts (hunk 1 above)
Comment (meganrogge) [reply src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatWidget.ts:143]: probably need the history stuff from that and maybe other things too?

Comment (meganrogge) [reply src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatWidget.ts:143]: I can reproduce that in insider's

Comment (meganrogge) [reply src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatWidget.ts:143]: Screenshot 2024-11-06 at 4 36 58 PM
that's still used

Comment (meganrogge) [reply src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatWidget.ts:143]: https://github.com/microsoft/vscode/blob/2cac90058f0df43c91dde33bb2b8910720529368/src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatActions.ts#L47

**Pull request closed without merging**